lambda-notes/src/components/CreateNote.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Button, Form, Label, Input, FormGroup } from 'reactstrap';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import {createNote} from '../actions';
+import '../css/CreateNote.css';
+
+
+
+class CreateNote extends Component {
+  constructor() {
+    super()
+    this.state={
+      title: '',
+      body: '',
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name] : event.target.value })
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.props.createNote(this.state);
+    this.setState({ title: '', body: ''});
```

Comment:
If you add `this.props.history.push('/')` at the end of the `handleSubmit` method, after creating a note, you'll be taken back to the view with the list of notes.